src/routes.tsx

```diff
@@ -24,6 +25,7 @@ export const routes = [
   { path: "/email/error", exact: true, component: EmailErrorPage },
   { path: "/news", exact: true, component: NewsPage },
   { path: "/news/:slug", exact: true, component: NewsPageDetail },
+  { path: "/learn", exact: true, component: LearnPage },
   { path: "/resources", exact: true, component: ResourcesPage },
```

Comment:
remove, and any other related which are no longer use

Reply:
Done.